Events/PinwheelErrorEvent.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  PinwheelErrorEvent.swift
+//  PinwheelSDK
+//
+//  Created by Robby Abaya on 2/12/21.
+//
+
+import Foundation
+
+public struct PinwheelErrorEvent: Codable {
```

Comment:
Not understanding the purpose of this data type? All events come back in this format not just error events.